support/cas-server-support-pac4j-api/src/main/java/org/apereo/cas/pac4j/TicketRegistrySessionStore.java

```diff
@@ -30,6 +30,21 @@
 @Slf4j
 @RequiredArgsConstructor
 public class TicketRegistrySessionStore implements SessionStore {
+    /**
+     * SAML server support prefix.
+     */
+    public static final String SAML_SERVER_SUPPORT_PREFIX = "SamlServerSupport";
+
+    /**
+     * OAuth/OIDC server support prefix.
+     */
+    public static final String OAUTH_OIDC_SERVER_SUPPORT_PREFIX = "OauthOidcServerSupport";
+
+    /**
+     * Authentication delegation prefix.
+     */
+    public static final String AUTHENTICATION_DELEGATION_PREFIX = "AuthnDelegation";
+
```

Comment:
Please remove these from there; they need to move to the modules that own them.

Reply:
In fact, I was inspired by this: https://github.com/apereo/cas/blob/6.6.x/support/cas-server-support-pac4j-api/src/main/java/org/apereo/cas/pac4j/DistributedJEESessionStore.java#L36
But yes, it would be better to have constants in their related modules.